modules/blocks.js

```diff
@@ -466,7 +467,7 @@ __private.applyBlock = function (block, broadcast, cb, saveBlock) {
 						err = ['Failed to apply transaction:', transaction.id, '-', err].join(' ');
 						library.logger.error(err);
 						library.logger.error('Transaction', transaction);
-						// TODO: Send a numbered signal to be caught by forever to trigger a rebuild.
+						// Fatal error, memory tables will be inconsistent
```

Comment:
Any time we kill lisk we should be explicit and say "Lisk has exited due to irrecoverable error" or something similiar